lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -2,5 +2,6 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
     expose :validate_import_vm
     expose :submit_import_vm
+    expose :submit_configure_imported_vm_networks
```

Comment:
@gmcculloug but maybe, since this is just an exposure of the provider method, the naming would  be better discussed in the dependent provider patch https://github.com/ManageIQ/manageiq-providers-ovirt/pull/47